editor/modes/visual-editor/block.js

```diff
@@ -30,7 +30,7 @@ import {
 	focusBlock,
 	insertBlocks,
 	mergeBlocks,
-	removeBlocks,
```

Comment:
Should we merge these two actions into one with `castArray`?

Reply:
This is what I was wondering too 😄 

>Consolidating `removeBlocks` and `removeBlock` into a single action creator?

Let's save the refactor for after.